raiden/storage/migrations/v19_to_v20.py

```diff
@@ -0,0 +1,168 @@
+import json
+
+from raiden.exceptions import RaidenUnrecoverableError
+from raiden.network.proxies.utils import get_onchain_locksroots
+from raiden.storage.sqlite import SQLiteStorage
+from raiden.utils.typing import Any, Dict
+
+RaidenService = 'RaidenService'
+
+SOURCE_VERSION = 19
+TARGET_VERSION = 20
+
+
+def _find_channel_new_state_change(
+        storage: SQLiteStorage,
+        token_network_address: str,
+        channel_identifier: str,
+):
+    return storage.get_latest_event_by_data_field({
+        '_type': 'raiden.transfer.state_change.ContractReceiveChannelNew',
+        'token_network_identifier': token_network_address,
+        'channel_state.identifier': channel_identifier,
+    })
+
+
+def _get_onchain_locksroots(
+        raiden: RaidenService,
+        storage: SQLiteStorage,
+        token_network: Dict[str, Any],
+        channel: Dict[str, Any],
+):
+    channel_new_state_change = _find_channel_new_state_change(
+        storage=storage,
+        token_network_address=token_network['address'],
+        channel_identifier=channel['identifier'],
+    )
+
+    if not channel_new_state_change:
+        raise RaidenUnrecoverableError(
+            'Could not find the state change for channel {channel_identifier}, '
```

Comment:
You are using `{}` in this string but forgot the `f string` prefix. I wonder if there is a tool that can warn us for this,